src/main/tables/70_container_images.sql

```diff
@@ -9,5 +9,6 @@ CREATE TABLE container_images (
   tag text NOT NULL,  -- tag used to pull down an image. We'll default it to 'latest'
   url text,           -- URL containing more information about the image (ex: docker hub URL)
   deprecated boolean NOT NULL DEFAULT FALSE,  -- flag indicating if tools using this image should be used in new apps.
+  osg_image_path varchar(1024), -- the path to the Singularity image in OSG's CVM-FS repository if available.
```

Comment:
Is this a known length limit for Singularity image paths?
Otherwise, I'd probably feel safer with a larger limit, or even just `text` 😁

Reply:
Heh, nope, it was just arbitrary. A text field probably is safer 😁